doc/api/events.md

```diff
@@ -302,6 +318,8 @@ Returns an array listing the events for which the emitter has registered
 listeners. The values in the array will be strings or Symbols.
 
 ```js
+'use strict';
+
```

Comment:
This example doesn't work because of a previous mistype (I will file a PR to fix it).

After fixing the mistype, this example works without `use strict`.